hooks/webs/useWebDetails.tsx

```diff
@@ -0,0 +1,28 @@
+import type { Location } from '@prisma/client'
+import { REMOTE_URL } from '@helpers/config'
+import { useQuery } from '@tanstack/react-query'
+
+export async function fetchWebDetailsRequest(isSSR = false) {
+  const response = await fetch(
+    isSSR ? `${REMOTE_URL}/api/web-details` : '/api/web-details',

```

Comment:
What does this line do?

Reply:
When on the server, it needs the full URL

Reply:
🤯  mind = blown